scipy/integrate/_ivp/tests/test_ivp.py

```diff
@@ -230,26 +230,26 @@ def test_integration_complex():
         assert_equal(res.status, 0)
 
         if method == 'DOP853':
-            assert_(res.nfev < 35)
```

Comment:
The fact that `test_integration_complex` worked with strangely evaluated "complex norms" is not ideal. Maybe we should add a more substantial test for integration in complex domain.

As for this PR --- looks good to me.